core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -869,10 +869,10 @@ public static function findMultipleByIds($arrIds, array $arrOptions=array())
 
 			if ($objMissing !== null)
 			{
-				while ($objMissing->next())
+				foreach ($objMissing as $objMissingRecord)
```

Comment:
Why did you choose `$objMissingRecord` instead of something short like `$model` or `$current` (or `$objModel` if you want to stick to the old naming schema)?

Reply:
I wanted to stick to the old naming schema and use a descriptive variable name. Since `$objMissing` is the collection then iterating over the collection gives you a single record - hence the name `$objMissingRecord`. Otherwise I would go for `$objCurrent`.